text/0037-minting.md

```diff
@@ -0,0 +1,167 @@
+* Nonce: random value to protect against replay attacks
+* Tombstone block: block index after which this transaction will be discarded by consensus, to prevent the transaction from being nominated indefinitely
+* The total amount that can be minted across all configurations included in the `MintConfigTx`
+* A list of configurations for this token id, each containing:
+    * Signer set: the set of _N_ signers, `{A_i, A_{i+1}, ..., A_N}`
+    * Threshold: the number, _M_, of signers required per `MintTx`
+    * Mint limit: the total amount that can be minted in this configuration
+* the Ed25519 multisig signature set, including the total _M_ signatures required for the _M-of-N_ threshold, specified by the governors configuration, previously signed by the Foundation minting trust root key.
+
+
+All of the values above are committed to the blockchain and will be used when verifying `MintTxs`.
+
+Consensus validator nodes perform consensus when a `MintConfigTx` is proposed via a client RPC endpoint on the nodes. This provides flexibility to the minting governors and keeps the network resilient so that a quorum must be reached to allow a new minting configuration, while ensuring liveness of the network, so that the network need not restart in order to change the minting configuration.
+
+Consensus validator nodes use the latest minting configuration to validate `MintTxs`.
+
+* Note, however, that nodes in catchup mode do not participate in consensus. The network is already resilient to nodes in catchup, and the minting procedures are similarly resilient.
+
+## Governors: The Signer Set for a Consensus Validator
+[governors]: #governors
+
+The consensus validator nodes must maintain a set of "master minters," also known as _governors_, who can sign the `MintConfigTx` transactions. The governor set is maintained in the node's startup configuration file, and it is the responsibility of the node operator to perform due diligence on any governor they are adding to the set.
+
+The governor set is a set of Ed25519 public keys and a threshold, and a `MintConfigTx`'s `TxPrefix` must be signed by the threshold number of governors required by each node in a quorum of nodes to establish a new Minting Configuration for the network.
+
+To further protect the governors configuration, and to ensure all nodes are running with identical configuration, the consensus enclave contains a baked-in Ed25519 public key called the `Minting Trust Root` key. The consensus enclave requires that the governors configuration be signed by the matching private key (held by the MobileCoin Foundation), and that the signature is provided upon initialization. Without a valid signature over the governors configuration, the consensus enclave will refuse to initialize.
+
+The governors configuration is also hashed into the node's responder ID, preventing node-to-node attestation if the configuration differs. This is similar to how minimum fee configuration is currently handled.
```

Comment:
ResponderId vs. NodeId is in a somewhat ambiguous state currently. Namely ResponderId just holds a uri and NodeId holds that + a signing key. Is this talking about hashing it into a NodeId?

Reply:
Responder Id is an attestation concept, Node Id is an SCP concept.